tempel.el

```diff
@@ -48,9 +48,9 @@
   :group 'editing
   :prefix "tempel-")
 
-(defcustom tempel-file (expand-file-name "templates" user-emacs-directory)
-  "Path to the template file."
-  :type 'string)
+(defcustom tempel-paths nil
+  "A file or a list of files and/or directories to look for snippets in."
+  :type '(choice (string list)))
```

Comment:
Better name. Call it `tempel-path` similar to `load-path`. And use `(choice string (repeat list))`.